packages/rfq-indexer/api/src/graphql/types/events.graphql

```diff
@@ -86,6 +86,6 @@ scalar BigInt
     blockNumber: BigInt!
     blockTimestamp: Int!
     transactionHash: String!
-    originChainId: Int!
-    originChain: String!
+    chainId: Int!
```

Comment:
Any reasoning behind this change? The only reason I did this was for standardization with all of our other queries (makes it neater to map fields). 

Not a big deal